### PR TITLE
add print_level option

### PR DESCRIPTION
**Pull request by zanellia**
1. adding print_level option
2. add opts_set setter to Python interface
3. changing to shorter setter name cons_set in Python interface

### CODE REVIEW

File: interfaces/acados_template/acados_template/AcadosOcpSolver.py [hunk 1]
```diff
@@ -539,7 +539,7 @@ def cost_set(self, stage_, field_, value_):
         return
 
 
-    def constraints_set(self, stage_, field_, value_):
+    def cons_set(self, stage_, field_, value_):
```
Comment (FreyJo) [inline interfaces/acados_template/acados_template/AcadosOcpSolver.py]: I think `constraints_set` is more clear and not too long.
If you really want to abbreviate it, I would use `constr` as in the Mex interface.
`cons` could be misinterpreted as constant.

Comment (zanellia) [reply interfaces/acados_template/acados_template/AcadosOcpSolver.py]: Fair enough. I like it better too, now that I think about it. Then let's make the convention that all setters must be <full_module_name>_set.

Comment (zanellia) [reply interfaces/acados_template/acados_template/AcadosOcpSolver.py]: @giaf?

**Comment by zanellia** [comment]
let's merge? :)

**Pull request merged**